source/sonata_config.rst

```diff
@@ -10,10 +10,29 @@ version
 
 *Optional*.
 
-An integer defining the version of the SONATA specification.
+A number defining the version of the SONATA specification.
 The default value is "1" as for the specification described in the SONATA original paper.
 This version is |CurrentSonataConfigVersion|.
 
+metadata
+--------
+
+Place for extra metadata about the circuit config.
+
+*Optional*.
+
+.. table::
+
+   =============================== ================ ====================================
+   Property                        Requirement      Description
+   =============================== ================ ====================================
+   status                          Optional         This describes how "complete" the circuit_config is:
+                                                     * `complete`: circuit is checked for all properties that aren't optional, and all fields should resolve.
+                                                     * `partial`: relaxes checking: `networks` may not exist, for instance; mandatory properties aren't enforced.
```

Comment:
Just wondering, for `partial`, the libsonata parser will not raise error ?

Reply:
Yeah, that's the idea - people want to have partial configs for circuit building, since it's convenient to pass around `circuit_config.json` that doens't have emodels yet (for instance)

Reply:
On that note, I think it would make sense that neurodamus failed if the config was marked partial.

Reply:
Good reminder. Better to make neurodamus fail or stop processing `partial` config at beginning, rather than fail later on because of missing components.

Reply:
Exactly - `Complete` means that all the required attributes/properties exist, so the config is good to be used.  Rejecting partial saves many checks later.